core/backend/package.json

```diff
@@ -92,9 +92,9 @@
   "dependencies": {
```

Comment:
could you also try updating the types/node dev dep to use ^ for all pkgs?

Reply:
Updated